src/Xamarin.Android.Build.Utilities/Sdks/MonoDroidSdkUnix.cs

```diff
@@ -22,6 +22,9 @@ class MonoDroidSdkUnix : MonoDroidSdkBase
 		protected override string FindRuntime ()
 		{
 			string monoAndroidPath = Environment.GetEnvironmentVariable ("MONO_ANDROID_PATH");
+			if (monoAndroidPath == null)
+				// for Mono.Posix and Mono.Data.Sqlite builds in xamarin-android.
+				monoAndroidPath = Path.GetFullPath (Path.Combine (new Uri (GetType ().Assembly.CodeBase).LocalPath, "..", "..", "..", "..", ".."));
```

Comment:
SearchPaths need to be absolute paths, which do not fit with this not-fixed path. Or do you mean something else?